tests/pool_test.go

```diff
@@ -634,3 +633,12 @@ func notDeletedVMs(poolName string, vms *v1.VirtualMachineList) (notDeleted []v1
 	}
 	return
 }
+
+func NotDeletedVMs(vms *v1.VirtualMachineList) (notDeleted []v1.VirtualMachine) {
```

Comment:
This can be private `notDeletedVMs`

Reply:
Right, but this file already have `notDeletedVMs()` func. So I left this one as is in this commit, and anyway it's deleted in a later commit.